tools/setup_helpers/dist_check.py

```diff
@@ -6,7 +6,7 @@
 from .cuda import USE_CUDA
 
 # On ROCm, RCCL development isn't complete. https://github.com/ROCmSoftwarePlatform/rccl
-USE_DISTRIBUTED = not check_negative_env_flag("USE_DISTRIBUTED") and not IS_WINDOWS and not check_env_flag("USE_ROCM")
+USE_DISTRIBUTED = not check_negative_env_flag("USE_DISTRIBUTED") and not IS_WINDOWS# and not check_env_flag("USE_ROCM")
```

Comment:
why just comment it? why not remove it and the comment above?

Reply:
I wasn't sure if I might have to keep USE_DISTRIBUTED disabled for ROCm by default. But based on our discussion, I'll go ahead and remove the commented part, and add skipIfRocms for the tests that fail with rccl build.